.github/workflows/job_python_unit_tests.yml

```diff
@@ -52,6 +52,13 @@ jobs:
           name: openvino_tests
           path: ${{ env.INSTALL_TEST_DIR }}
 
+      - name: Download OpenVINO tokenizers extension
+        if: ${{ runner.os != 'macOS' && runner.arch != 'ARM64' }} # Ticket: 126287
+        uses: actions/download-artifact@v3
+        with:
+          name: openvino_tokenizers_wheel
+          path: ${{ env.INSTALL_DIR }}
```

Comment:
@akashchi can we extract building tokenizers on Unix as a separate workflow and reuse in macOS / Linux both arm64 / x86_64 ?

It will allow:
- increase a range of supported / tested platforms
- remove conditional code below when installing tokenizers

Reply:
Created tickets to be addressed this